plugin/cypress/README.md

```diff
@@ -12,9 +12,9 @@ Before you start using Cypress suite, you might need export some environment var
 
 ```bash
 export CYPRESS_BASE_URL=<value>               # defaults to http://localhost:9000
-export CYPRESS_OC_CLUSTER_USER=<value>        # defaults to jenkins, opt. kubeadmin
-export CYPRESS_OC_CLUSTER_PASS=<value>        # no defaults
-export CYPRESS_OC_IDP=<value>                 # defaults to my_htpasswd_provider
+export CYPRESS_USERNAME=<value>               # defaults to jenkins, opt. kubeadmin
```

Comment:
Originally I chose different variable names to be double sure that settings from kiali/kiali will not interfere with OSSMC suite